sdk/schemaregistry/azure-schemaregistry-avroencoder/README.md

```diff
@@ -254,7 +261,7 @@ with eventhub_consumer, avro_encoder:
 
 ### General
 
-Azure Schema Registry Avro Encoder raises exceptions defined in [Azure Core][azure_core].
+Azure Schema Registry Avro Encoder raises exceptions defined in [Azure Core][azure_core] if errors are encountered when communicating with the Schema Registry service. Errors related to invalid content and content types will be raised as `azure.schemaregistry.encoder.avroencoder.InvalidContentError`, where `__cause__` will contain the underlying exception raised by the Apache Avro library. Errors related to invalid schema will be raised as `azure.schemaregistry.encoder.avroencoder.InvalidSchemaError`, where `__cause__` will contain the underlying exception raised by the Apache Avro library.
```

Comment:
since __cause__ is the same for both, do you think making that explanation separate sentence less repetitive or would it be more confusing?